bpf/kmesh/workload/xdp.c

```diff
@@ -122,6 +186,15 @@ int xdp_shutdown(struct xdp_md *ctx)
 
     // never failed
     parser_tuple(&info, &tuple_info);
+#ifdef AUTH_BY_XDP
+    // Before the authentication types supported by eBPF XDP are fully implemented,
+    // this section only processes AUTH_DENY. If get AUTH_ALLOW,
+    // it will still depend on the user-space authentication process to match other rule types.
+    if (xdp_rbac_manage(&info, &tuple_info) == AUTH_DENY) {
```

Comment:
what's the different between AUTH_DENY and AUTH_FORBID?

Reply:
AUTH_DENY for xdp-match auth in bpf， AUTH_FORBID for sockets-match-auth in user mode